ontologies/semopenalex-ontology.ttl

```diff
@@ -0,0 +1,527 @@
+  rdfs:range xsd:float .
+
+gn:long a owl:DatatypeProperty;
+  rdfs:domain <https://semopenalex.org/class/Geo>;
+  rdfs:range xsd:float .
+
+<https://dbpedia.org/ontology/alternativeName> a owl:DatatypeProperty;
+  rdfs:domain <https://semopenalex.org/class/Institution>;
+  rdfs:range xsd:string .
+
+<https://dbpedia.org/property/acronym> a owl:DatatypeProperty;
+  rdfs:domain <https://semopenalex.org/class/Institution>;
+  rdfs:range xsd:string .
+
+<https://semopenalex.org/property/imageThumbnail> a owl:DatatypeProperty;
+  rdfs:domain _:node110;
+  rdfs:range xsd:string .
+
+foaf:depiction a owl:DatatypeProperty;
+  rdfs:domain _:node113;
+  rdfs:range xsd:string .
+
+<https://semopenalex.org/property/rorType> a owl:DatatypeProperty;
+  rdfs:domain <https://semopenalex.org/class/Institution>;
+  rdfs:range xsd:string .
+
+<https://dbpedia.org/property/countryCode> a owl:DatatypeProperty;
+  rdfs:domain <https://semopenalex.org/class/Institution>;
+  rdfs:range xsd:string .
+
+<https://semopenalex.org/property/grid> a owl:DatatypeProperty;
+  rdfs:domain <https://semopenalex.org/class/Institution>;
+  rdfs:range xsd:string .
+
+<https://semopenalex.org/property/ror> a owl:DatatypeProperty;
+  rdfs:domain <https://semopenalex.org/class/Institution>;
+  rdfs:range xsd:string .
+
+<https://semopenalex.org/property/year> a owl:DatatypeProperty;
+  rdfs:domain <https://semopenalex.org/class/CountsByYear>;
+  rdfs:range xsd:integer .
+
+<https://semopenalex.org/property/alternativeName> a owl:DatatypeProperty;
+  rdfs:domain <https://semopenalex.org/class/Author>;
+  rdfs:range xsd:string .
+
+<https://dbpedia.org/property/twitter> a owl:DatatypeProperty;
+  rdfs:domain <https://semopenalex.org/class/Author>;
+  rdfs:range xsd:string .
+
+<https://dbpedia.org/property/scopus> a owl:DatatypeProperty;
+  rdfs:domain <https://semopenalex.org/class/Author>;
+  rdfs:range xsd:string .
+
+<https://dbpedia.org/ontology/orcidId> a owl:DatatypeProperty;
+  rdfs:domain <https://semopenalex.org/class/Author>;
+  rdfs:range xsd:string .
+
+<https://semopenalex.org/property/level> a owl:DatatypeProperty;
+  rdfs:domain <https://semopenalex.org/class/Concept>;
+  rdfs:range xsd:integer .
+
+<https://semopenalex.org/property/umlsAui> a owl:DatatypeProperty;
+  rdfs:domain <https://semopenalex.org/class/Concept>;
+  rdfs:range xsd:string .
+
+<https://semopenalex.org/property/umlsCui> a owl:DatatypeProperty;
+  rdfs:domain <https://semopenalex.org/class/Concept>;
+  rdfs:range xsd:string .
+
+<https://semopenalex.org/property/score> a owl:DatatypeProperty;
+  rdfs:domain <https://semopenalex.org/class/ConceptScore>;
+  rdfs:range xsd:integer .
+
+rdf:type a owl:ObjectProperty;
+  rdfs:domain <https://semopenalex.org/class/Work>;
+  rdfs:range <https://api.crossref.org/types> .
+
+dcterms:creator a owl:ObjectProperty;
+  rdfs:domain <https://semopenalex.org/class/Work>;
+  rdfs:range <https://semopenalex.org/class/Author> .
+
+dcterms:title a owl:DatatypeProperty;
```

Comment:
why are you reclaring properties from dcterms? Is this aligned with @phaasenase ?